tests/provision/reboot/test.sh

```diff
@@ -27,6 +27,22 @@ rlJournalStart
         rlPhaseEnd
     fi
 
+    if [[ "$PROVISION_METHODS" =~ beaker ]]; then
+        rlPhaseStartTest "Beaker"
+            rlRun "tmt run --scratch -i $run provision -h beaker -i Fedora-39"
```

Comment:
What about not specifying the image and going with the default one? We wouldn't have to update this regularly in the future.

Reply:
That didn’t work for me with Beaker, I was gettin “bo such distro” errors :/

Reply:
Hm, interesting, for me Beaker was unhappy with `Fedora-39` :-) What about `--image fedora`? Would that work for you?

Reply:
For me? Sure. For Beaker? I don’t know, what you suggest is the default that did not work for me today :) Maybe I’m missing some piece of configuration, who knows. I’ll give it a try tomorrow - I didn’t pay too much attention to it today.

Reply:
Strange. Could it be that your `/etc/tmt/mrack-provisioning-config.yaml` does not have effect? Or you have an old one? But still, even with the old mapping to `Fedora-37` it works for me... Don't know.

Reply:
Agreed on chat to use the default image: c421c2b5